lineman/app/services/lmo_url_service.coffee

```diff
@@ -12,6 +12,9 @@ angular.module('loomioApp').factory 'LmoUrlService', (AppConfig) ->
     routePath: (route) ->
       "/".concat(route).replace('//', '/')
 
+    membershipRequest: (mr, params = {}, options = {}) ->
+      @buildModelRoute('g', mr.group().key, 'membership_requests', params, options)
```

Comment:
oh cool. I really like the way that reads. thank you!

On Fri, 23 Oct 2015 at 19:49 James Kiesel <notifications@github.com> wrote:

> In lineman/app/services/lmo_url_service.coffee
> <https://github.com/loomio/loomio/pull/2635#discussion_r42837291>:
>
> > @@ -12,6 +12,9 @@ angular.module('loomioApp').factory 'LmoUrlService', (AppConfig) ->
> >      routePath: (route) ->
> >        "/".concat(route).replace('//', '/')
> >
> > +    membershipRequest: (mr, params = {}, options = {}) ->
> > +      @buildModelRoute('g', mr.group().key, 'membership_requests', params, options)
>
> Hm, I'm surprised it's not stubbing 'membership_request' to
> 'membership-request'
>
> We could use the 'route' method here, which is what we use for lmo-href-for
>
> membershipRequest: (mr, params = {}, options = {}) ->
>   @route model: mr.group(), action: 'membership_requests', params: params
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/2635/files#r42837291>.
>